src/scaladoc/scala/tools/nsc/doc/base/CommentFactoryBase.scala

```diff
@@ -254,16 +254,16 @@ trait CommentFactoryBase { this: MemberLookupBase =>
         else lastTagKey match {
           case Some(key) =>
             val value =
-              ((tags get key): @unchecked) match {
+              (tags get key: @unchecked) match {
                 case Some(b :: bs) => (b + endOfLine + marker) :: bs
                 case None => oops("lastTagKey set when no tag exists for key")
               }
-            parse0(docBody, tags + (key -> value), lastTagKey, ls, inCodeBlock = true)
+            parse0(docBody, tags ++ Map(key -> value), lastTagKey, ls, inCodeBlock = true)
```

Comment:
I don't think this needs to be wrapped in a `Map`

Reply:
I added this to react to “+” being a deprecated function in 2.13

I tried using other ways to add this key value pair to the mutable map, but it seemed to me that this was prevented due to the @unchecked nature of tags (not sure)

I will give that another try and see if I can find a more elegant solution to replace “.+”

Reply:
I changed these code parts to instead use an immutable map for the whole pattern matching clause and the usage of tags in this function to react to the deprecation of "+" on the generic Map trait. See my latest commit on the PR branch for this

Is that a valid solution for you?

Reply:
Yup, sounds good. `+` always creates a new `Map` regardless of whether or not it's mutable, so it may as well be immutable (which is at least optimised for such updates).